dask_bigquery/core.py

```diff
@@ -196,13 +238,19 @@ def make_create_read_session_request():
                 ),
             )
 
+        arrow_options_meta = arrow_options.copy()
+        if pyarrow_strings_enabled():
+            types_mapper = _get_types_mapper(arrow_options.get("types_mapper", {}.get))
+            if types_mapper is not None:
+                arrow_options_meta["types_mapper"] = types_mapper
```

Comment:
We have this twice, once here and once in `bigquery_read`. Thoughts on keeping this here, passing `arrow_options` (with the correct `types_mapper`) through to `dd.from_map` below? That way we could drop the `convert_string=` parameter.

Reply:
I thought about this too.

I'd like to avoid serialising that stuff in the graph, just passing a flag seems a lot easier.

Reply:
Ah, I see. Are the `types_mapper` entries we're adding particularly large?

Reply:
It adds a few callables, which doesn't seem like a good idea (didn't do any profiling)